Add in the Trident brand/logo files for the Lua loader.

beanpole135 opened the pull request.

beanpole135 committed:
Add in the logo/brand files for Trident in the Lua loader.

beanpole135 committed:
Oops, forgot to change the copyright in the new trident branding files.

beanpole135 committed:
Merge branch 'trueos-master' into trident-brand-lua

kmoore134 commented on stand/lua/brand-trident.lua:
```diff
@@ -0,0 +1,46 @@
+local trident_brand = {
+"     _____     _     _            _       ",
+"    |_   _| __(_) __| | ___ _ __ | |_     ",
+"      | || '__| |/ _` |/ _ \ '_ \| __|    ",
```
This is going to fail to load. You need to escape the \ characters, so they are \\. Otherwise you'll get LUA errors on import. Take a look at the red-highlighting here on github to see what I mean.

beanpole135 committed:
Update brand-trident.lua

Make sure all the "\" character in the branding are doubled up (escape character for lua)
